src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
-export interface Figure {
-
+function errorParams(...args: number[]): boolean {
+  return args.some((el: number): boolean => {
```

Comment:
`el` or `res` is not a good name for the variable. Better name `value` for example.